velox/dwio/parquet/writer/arrow/Statistics.cpp

```diff
@@ -94,9 +97,15 @@ struct CompareHelper {
       "T is an unsigned numeric");
 
   constexpr static T DefaultMin() {
+    if constexpr (std::is_floating_point_v<T>) {
```

Comment:
I would suggest to add the INF process to GetMinMax, the DefaultMin is Inf looks not reasonable.